api/main.py

```diff
@@ -135,8 +129,8 @@ def get_devices_for_house(house_id, token):
 
 
 @api.route('/device/<string:device_id>')
```

Comment:
All routes with authentication should be POST routes, as we're *posting* the token to the API. Can you change this for all routes, please?